test/e2e/type-module-interop/index.test.ts

```diff
@@ -12,10 +12,24 @@ describe('Type module interop', () => {
       files: {
         'pages/index.js': `
           import Link from 'next/link'
+          import Head from 'next/head'
+          import Script from 'next/script'
 
           export default function Page() { 
             return (
               <>
+                <Head>
+                  <title>This page has a title 🤔</title>
+                  <meta charSet="utf-8" />
+                  <meta name="viewport" content="initial-scale=1.0, width=device-width" />
+                  <Script
```

Comment:
`next/script` doesn't need to be nested inside of `next/head` so we can move it out ([related doc](https://nextjs.org/docs/basic-features/script))